.github/workflows/status-check-no-build.yaml

```diff
@@ -0,0 +1,14 @@
+name: Microsoft Graph CLI - Release
+
+on:
+  pull_request:
+    paths-ignore:
+    - .azure-pipelines
+    - src
```

Comment:
I'm curious about this entry, isn't that going to disqualify most of the code?

Reply:
I'll convert it to a draft so you don't get notified

Reply:
I'm not sure I understand the either or need here? We always need to run github actions. And we only need to run ADO when releasing because some of the tasks are not available on github actions yet. The minute they becomes available on github actions, we'll get off ADO.

Reply:
It hasn't worked. I'm giving up.

Reply:
> I'm not sure I understand the either or need here? We always need to run github actions. And we only need to run ADO when releasing because some of the tasks are not available on github actions yet. The minute they becomes available on github actions, we'll get off ADO.

It was about the branch protection rules. I added a branch protection that the CI has to pass, but I wanted to skip building the CLI if files that don't affect the binary change. Problem is, the branch protection rule would then be stuck waiting for a status check.